test/xspec.bats

```diff
@@ -190,3 +190,10 @@ teardown() {
     [ "$status" -eq 0 ]
     [[ "${lines[3]}" =~ "Testing with" ]]
 }
+
+@test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	run bash -c "java -Xmx1024m -cp ${XMLCALABASH_CP} com.xmlcalabash.drivers.Main -isource=xspec-72.xspec xspec-home=file:///home/travis/build/xspec/xspec/ -oresult=/home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html /home/travis/build/xspec/xspec/src/harnesses/saxon/saxon-xslt-harness.xproc && grep 'charset=UTF-8' /home/travis/build/xspec/xspec/test/xspec/xspec-72-report.html"
```

Comment:
Yes, I think that's an alternative approach. However, you would still need to chain the two commands (I used ```&&``` in the bats test) or execute them within the same test as the test must check that the HTML contains ```charset=UTF-8```. 

I feel that that the ```grep``` command is shorter and is also very common in Linux. I saw that in [xspec-bat.cmd](https://github.com/xspec/xspec/blob/master/test/xspec-bat.cmd) you used ```findstr``` which I think is the equivalent of ```grep``` in Windows. Would that work for implementing this test?

Let me know if I misunderstood your comment.